apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Select_NonNumeric_BeneAndClaimIds.sql

```diff
@@ -0,0 +1,65 @@
+--
+-- Script to locate all records that have non-numeric bene_id's
+-- or clm_id's in preparation for the conversion of bene_id from
+-- varchar to bigint. All of these queries should return zero
+-- on all environments as a precondition to that migration.
+--
+
+-- Find non-numeric bene_id's and clm_id's in claim and claim_lines tables
+SELECT clm_id FROM carrier_claim_lines
+WHERE clm_id IN (SELECT clm_id FROM carrier_claims WHERE bene_id !~ E'^([-+])?[0-9]+$' OR clm_id !~ E'^([-+])?[0-9]+$');
```

Comment:
🍰  Should we maybe DECLARE a variable for the pattern here and just ref it in the queries? Will be easier to edit if we need to make adjustments.

Reply:
I like the idea however this is a one-off script that is just being reviewed and merged for historical purposes so I'm not planning on making the change.